BGL/include/CGAL/boost/graph/properties.h

```diff
@@ -227,7 +305,54 @@ void init_halfedge_indices(PolygonMesh& pm, HalfedgeIndexMap hid)
                               >::type >::type() );
 }
 
-} } //end of namespace CGAL::helpers
+} //namespace helpers
+
+
+namespace internal {
+
+template <typename T, typename G>
+typename boost::dynamic_property_map<G,boost::vertex_property_t<T> >::const_type
+add(boost::vertex_property_t<T> prop, const G& g)
```

Comment:
The pollution of the `boost` namespace is `dynamic_property_map`.

`add` and `remove` are added to `CGAL::internal`.

> get it into the release
About that: delay the PR to CGAL-4.12 is still the option I prefer. I follow you in the attempt to get it into CGAL-4.11, but that is not my choice.